### PR TITLE
Added pause and community pause

### PR DESCRIPTION
**Pull request by shivam-ef**

### CODE REVIEW

File: src/LrtSquare.sol [hunk 1]
```diff
@@ -356,6 +379,47 @@ contract LrtSquare is
...
+        communityPauseDepositedAmt = msg.value;
+        emit CommunityPause(msg.sender);
+    }
+
+    function pause() external onlyRole(PAUSER_ROLE) whenNotPaused {
+        _pause();
+    }
+
+    function unpause() external onlyRole(PAUSER_ROLE) whenPaused {
+        if (communityPauseDepositedAmt != 0)
```
Comment (seongyun-ko) [inline src/LrtSquare.sol:397]: can't it unpause when paused by `pause` instead of `communityPause`?

**Review by seongyun-ko — COMMENTED** [review]
how should we pause the actions associated with a certain token? 
for example, we found an issue with a token `X`, how to pause its deposit?

File: src/LrtSquare.sol (hunk 1 above)
Comment (Shivam78288) [reply src/LrtSquare.sol:397]: The pauser would have to withdraw the community pause deposit amount from the contract before they unpause

**Review by seongyun-ko — APPROVED** [review]
lgtm

**Pull request merged**